changelog/unreleased/pull-2906

```diff
@@ -0,0 +1,9 @@
+Bugfix: Fix inconsistent timestamps and permissions on directories restored
+
+We've fixed the inconsistent timestamps and permissions on directories not
```

Comment:
I'd include a more explicit description of the solved problem, maybe something like this:
```
When using the `--include` option with the restore command, it only restored
timestamps and permissions on directories selected by the include pattern.
Intermediate directories which are necessary to restore files located in
subdirectories, were created with default permissions. We've fixed the
`restore` command to also restore permissions for these directories.
```

Reply:
done, replace with this one